src/Telegram.Bot/Requests/Games/SendGameRequest.cs

```diff
@@ -22,6 +22,12 @@ public class SendGameRequest : RequestBase<Message>, IChatTargetable
     /// <inheritdoc />
     ChatId IChatTargetable.ChatId => ChatId;
```

Comment:
Shoul it be `new (ChatId)` like the other *GameRequest classes?

Reply:
No, it was a mistake calling new in the other request since I kept implicit conversion from ChatId to long. Though maybe we should drop it too. What do you think?

Reply:
As much as I would love to remove it, this change might annoy a lot of folks out there and require too many changes. 

If we do it, we should use ChatId (or some derived class) in all relevant cases as a domain-specific identity type (linked_chat_id, Chat.Id, User.Id, Message.migrate_to_chat_id, Message.migrate_from_chat_id, Contact.user_id, etc..). 
They all represent the same entity and having to instantiate `new ChatId(contact.UserId)` to send message to user would be unintuitive.